src/Columns/ColumnNullable.cpp

```diff
@@ -27,6 +27,11 @@ namespace ErrorCodes
     extern const int NOT_IMPLEMENTED;
 }
 
+template <typename T>
+inline bool contain_byte(const T* __restrict data, const size_t length, const signed char byte)
```

Comment:
Maybe memoryIsZero has better performance, should check it. And later memoryIsZero will be replaced by more higher performance function.

Reply:
std::memchr is 1.8x faster than memoryIsZero, but is slower than is 40% isa-l isisal_zero_detect.